web.d.ts

```diff
@@ -1,11 +1,35 @@
 /// <reference types="vite/client" />
 
-// This file must have at least one export or import on top-level
-import { Ability, ClientService, LoadingService, PreviewService } from 'web-pkg'
+import { Ability, ClientService, LoadingService } from 'web-pkg'
+import { OwnCloudSdk } from 'web-client/src/types'
+import { Language } from 'vue3-gettext'
+import { Router, RouteRecordNormalizedLoaded } from 'vue-router'
+import { Store } from 'vuex'
+import { ClientService, LoadingService, PreviewService } from 'web-pkg/src/services'
+import { UppyService } from 'web-runtime/src/services/uppyService'
+import { ThumbnailService } from 'web-app-files/src/services'
```

Comment:
Is it fine to import from the files app here? 🤔

Reply:
oh hehe, the import itself is outdated obviously - it's called preview service and lives in web-pkg as you might know 😅 

I apparently messed up the rebase ... very good catch